charts/tidb-cluster/values.yaml

```diff
@@ -33,6 +33,9 @@ pd:
   # refer to https://kubernetes.io/docs/concepts/storage/storage-classes
   storageClassName: local-storage
 
+  # Image pull policy.
+  imagePullPolicy: Always
```

Comment:
We'd better use `IfNotPresent` as default value.

Reply:
all image's pull policy modify to default ```IfNotPresent ```